force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.html

```diff
@@ -8,7 +8,6 @@
                     label={labelReference.settingsButtonEdit}
                     title={labelReference.settingsButtonEdit}
                     onclick={handleEditClick}
-                    class="slds-p-right_x-small"
```

Comment:
removing these because based on the mocks, buttons should be aligned with affordance.

Reply:
Done! It looks like the sizes of cancel and save are different. I left them as they were since the mocks was showing the same way.